rancher_pr_deploy.sh

```diff
@@ -17,25 +23,35 @@ if [[ -n "$TRAVIS" ]]; then
   # IF PULL REQUEST
   if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
 
-    install_rancher
+    if [ ! "$(command -v rancher)" ]
+    then 
+      install_rancher
+    fi
 
-    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/${RANCHER_TEMPLATE_NAME}/0/docker-compose.yml"  -o docker-compose.yml
-    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/${RANCHER_TEMPLATE_NAME}/0/rancher-compose.yml" -o rancher-compose.yml
+    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/feature/changes-for-ci-script/templates/${RANCHER_TEMPLATE_NAME}/0/docker-compose.yml"  -o docker-compose.yml
```

Comment:
This is hard coded for the feature branch for testing will need to be changed once the template changes are merged.